### PR TITLE
Add content to the README

### PR DESCRIPTION
**Pull request by jackzampolin**
A stab at a `README` cc @alexanderbez @AlexeyAkhunov

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,39 @@
-# Ethermint Skeleton
+[![](https://godoc.org/github.com/cosmos/ethermint?status.svg)](http://godoc.org/github.com/cosmos/ethermint)  [![Go Report Card](https://goreportcard.com/badge/github.com/cosmos/ethermint)](https://goreportcard.com/report/github.com/cosmos/ethermint)
 
-To build, execute these two commands:
+# Ethermint
 
-dep ensure
+__**WARNING:**__ Ethermint is under VERY ACTIVE DEVELOPMENT and should be treated as pre-alpha software. This means it is not meant to be run in production, its APIs are subject to change without warning and should not be relied upon, and should it be used to hold any value. We will remove this warning when we have a release that is stable, secure, and properly tested.
```
Comment (jbibla) [inline README.md]: "and should _not_ be used to hold any value"

**Review by alexanderbez — CHANGES_REQUESTED** [review]
Looks great! Just the one comment that @jolesbi left :-)

File: README.md [hunk 2]
```diff
@@ -1,9 +1,39 @@
...
+### Building Ethermint
+
+To build, execute the following commands:
+
+```bash
+# To build the binary and put the results in ./build
+$ make get-tools get-vendor-deps build
+
+# To build the project and install it in $GOBIN
+$ make get-tools get-vendor-deps install
```
Comment (zramsay) [inline README.md]: this is about to change following convo with @alexanderbez

Comment (jackzampolin) [reply README.md]: What does that mean?

Comment (alexanderbez) [reply README.md]: the targets will change: `make get-tools deps install`.

File: README.md (hunk 1 above)
Comment (jackzampolin) [reply README.md]: Wow that is the EXACT OPPOSITE of what I meant to say. Thanks for the catch there

**Comment by jackzampolin** [comment]
Made the fix here!

**Comment by alexanderbez** [comment]
@jack do we wanna update the make target names here too or wait until they're actually changed?

**Comment by jackzampolin** [comment]
I've gone ahead and merged `master` back in here and then updated the make targets to match https://github.com/cosmos/ethermint/issues/424

**Review by alexanderbez — APPROVED** [review]
LGTM 👍

**Pull request merged**